io/StacReader.cpp

```diff
@@ -411,7 +428,7 @@ void StacReader::initializeItem(NL::json stacJson)
 
     NL::json asset = stacJson.at("assets").at(assetName);
 
-    std::string dataUrl = asset.at("href").get<std::string>();
+    std::string dataUrl = getPath(itemPath, asset.at("href").get<std::string>());
```

Comment:
Please break this into two lines and add a comment about what we're doing. It reads a bit weird – we're getPath'ing an href – why?